app/grandchallenge/evaluation/models.py

```diff
@@ -134,6 +134,14 @@
     },
 }
 
+SELECTABLE_GPU_TYPES_SCHEMA = {
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "array",
+    "title": "The Selectable GPU Types Schema",
+    "items": {"type": "string", "required": [], "properties": {}},
+    "uniqueItems": True,
+}
```

Comment:
We can make the schema stronger by integrating the options (this is untested! so we should add a test to make sure that providing a wrong selection is caught):

```suggestion
SELECTABLE_GPU_TYPES_SCHEMA = {
    "$schema": "http://json-schema.org/draft-07/schema",
    "type": "array",
    "title": "The Selectable GPU Types Schema",
    "items": {
        "type": "string",
        "enum": [choice.value for choice in GPUTypeChoices]
    },
    "uniqueItems": True
}
```

Reply:
Excellent! Already tested manually and this works as expected so I remove the manual validation  in the clean method. I'll write a test for it now.